statisticsBot/code.js

```diff
@@ -1,48 +1,75 @@
 // Url to google spreadsheet contaning PNCSUP issues statistics
 spreadsheetUrl = 'https://docs.google.com/spreadsheets/d/1U5hMgVopnEmz6RNCRYSwMKlVtp2oW8r7XtHna9OhS58/edit#gid=1277613764';
+// spreadsheetUrl = 'https://docs.google.com/spreadsheets/d/17D6i87fPvf9Rx3UCWxMxkq3Qpa_us15dbCQl8bQZEP8/edit#gid=0'; // Testing sheet
 googleChatUrl = 'https://chat.google.com/';
+jiraSupportBoardUrl = 'https://projects.engineering.redhat.com/browse/';
```

Comment:
I think we are not allowed to leak internal URLs to the public

Reply:
Should I remove it just from git or try to integrate some config file into google scripts?

Reply:
Whatever is easier for you! But I guess the latter is cleaner. Thanks!